src/deriveFormData.ts

```diff
@@ -52,6 +53,9 @@ export const deriveFormData = memoize((schema: any, data: any, selection: string
         } else {
           // There's one edge case where path = ""
           output[key] = path === '' ? data : get(data, path);
+          if (output[key] === undefined && resolver) {
+            output[key] = resolver(path.split('.'));
```

Comment:
Seems like `resolver` should accept an array of numbers and/or strings.

Reply:
all it ever gets is the output of `.split('.')` so....... `resolver` doesn't have to support numbers. I think if your resolver _did_ support numbers that wouldn't be a type error right? E.g. if

```ts
const myResolver: (path: Array<number | string>) => any
```

that would still work right?

Reply:
ayayayaya nvm it all makes sense now